docs/modules/ROOT/pages/references/services-vshn.adoc

```diff
@@ -42,3 +42,28 @@ type:: bool
 default:: `true`
 
 Whether to enable NetworkPolicy in the PostgreSQL instance namespace. This allows to establish connections from claim's namespace to the PostgreSQL instance.
+
+
+=== `postgres.bucket_region`
+type:: string
+default:: `''`
+
+Bucket region, required for xObjectBucket and PostreSQL managed by VSHN backup, should be cloud specific.
+
+.Examples
+* Cloudscale
+** 'lpg'
+* Exoscale
+** 'ch-gva-2' 
+
+=== `postgres.bucket_endpoint`
+type:: string
+default:: `''`
```

Comment:
Maybe we want a default here @Kidswiss ? I know that in some cases we used Cloudscale as default.

Reply:
The default should be whatever the underlying cloud is.

E.g. APPUiO Cloudscale -> cloudscale
APPUiO AWS -> aws
etc.

Not sure if we can do dynamic facts as defaults?

Reply:
I see we have cloud in the [facts](https://syn.tools/commodore/reference/parameters.html#_facts). @wejdross you may try to use this as default.

Reply:
@glrf can You put an input here, because as I spoke with You about that, You pointed out that might not be the best idea to go with facts::cloud

Reply:
We can't find the bucket endpoint from the facts, we'd need to hardcode bucket endpoints per cloud. Then deploying it to a new cloud would involve adding new hardcoded endpoints.

IMO the way we currently do it is more generic and not really more effort. Once we have appcat on more than three clusters such config can be added to the global defaults.

Reply:
@wejdross please next time just add a comment earlier as to why the idea we discussed (me and @Kidswiss ) is not the best way to move forward. I feel this discussion was dragged too long here.